zproject/prod_settings_template.py

```diff
@@ -266,6 +266,22 @@
 #    ]
 # }
 
+
+## LDAP group sync configuration.
+## See: https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#synchronizing-groups
+# AUTH_LDAP_GROUP_SEARCH = LDAPSearch(
+#     "ou=groups,dc=www,dc=example,dc=com", ldap.SCOPE_SUBTREE, "(objectClass=groupOfUniqueNames)"
+# )
+
+# LDAP_GROUP_SYNC_LIST = {
+#   "subdomain1" : [
+#       "group1",
+#       "group2",
+#   ]
+# }
```

Comment:
What do you think about calling this `LDAP_SYNCHRONIZED_GROUPS_BY_REALM`? I don't like the `_LIST` suffix since it's actually a dictionary; and maybe it's worth having the `_BY_REALM` suffix just to make it easier to guess its structure.

Reply:
see 84f904fde9a17984662c645ecb6a4e6be76524ec